package.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <package format="2">
   <name>ros_tcp_endpoint</name>
-  <version>0.4.0</version>
+  <version>0.5.0</version>
```

Comment:
do we only upgrade the package version when merging from `dev` to `main`?

Reply:
We're about to release it this week, so it doesn't seem worth being nitpicky about it :-)

Reply:
Haha, sure; it is not important.